### PR TITLE
add summary for content

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

File: ApiBundle/Controller/ContentTypeController.php [hunk 1]
```diff
@@ -74,6 +76,25 @@ public function listAction(Request $request)
     }
 
     /**
+     * @param Request $request
+     *
+     * @Config\Route("/content/content-type-list", name="open_orchestra_api_content_type_list_for_content")
+     * @Config\Method({"GET"})
+     * @return FacadeInterface
+     */
+    public function listForContentAction(Request $request)
```
Comment (alavieille) [inline ApiBundle/Controller/ContentTypeController.php]: can you add a functional test for this method

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Content/ContentRouter.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import OrchestraRouter    from '../OrchestraRouter'
+import app                from '../../Application'
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Content/ContentRouter.js]: use Application

File: BackofficeBundle/Resources/public/template/Content/summaryElementView._tpl.html [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<div class="col-lg-6">
+    <div class="block">
+        <span><%= name %></span>
+        <ul class="action-links">
+            <li><a href="#">Lister</a></li>
+            <li><a href="#">Ajouter</a></li>
+        </ul>
+    </div>
+</div>
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/template/Content/summaryElementView._tpl.html]: eof

**Comment by itkg-nanne** [comment]
update done @alavieille

**Pull request merged**